chapters/en/chapter10/1.mdx

```diff
@@ -0,0 +1,19 @@
+# Introduction to Argilla[[introduction-to-argilla]]
+
+In Chapter 5 you learnt how to build a dataset using the 🤗 Datasets library and in Chapter 6 you explored how to fine-tune models for some common NLP tasks. In this chapter, you will learn how to use Argilla to **annotate and curate datasets** that you can use to train and evaluate your models.
```

Comment:
```suggestion
In Chapter 5 you learnt how to build a dataset using the 🤗 Datasets library and in Chapter 6 you explored how to fine-tune models for some common NLP tasks. In this chapter, you will learn how to use [Argilla](https://argilla.io) to **annotate and curate datasets** that you can use to train and evaluate your models.
```

Probably good to link Argilla at the first instance so that people can check it out.